### ISSUE TITLE
Can't Synchronize Account

### ISSUE THREAD

**Issue by erichserrano** [mycelium-com/wallet-ios#53]
My bitcoins are completely useless now because the account can't synchronize and I cant send my bitcoins out from my wallet. All my coins are stuck in this mycelium wallet and i have no way of sending them to a safer wallet that actually works. PLEASE DO NOT USE THIS APP! You will lose your coins and have NO way of getting them back. People have been complaining about this issue for weeks and it has yet to be resolved. Mycelium as a company clearly does not care for its users. I can't believe i put my trust in this company and in this app. My coins are stuck in their wallet and i cant get them back i am completely devastated. I hope this is eventually resolved. Worst bitcoin wallet ever 👎

**Comment by kumar45e** [comment]
Is it solved?

**Comment by erichserrano** [comment]
Yes I was able to get my coins into a better more supported wallet. Thank
you for finally reaching out to me.

**Comment by thorbylorbar** [comment]
How did you go about it please 
I’m having the same problem

**Comment by erichserrano** [comment]
I had to keep going on the app a couple times a day for a few weeks until
it finally worked and I quickly transferred my coins to a new wallet

**Comment by dadelgadillo96** [comment]
hello my wallet did not serve to send said that I did not synchronize I did the back up and saved my words now I enter again with my back up and I do not see my 18 mbtc that I had I want to know how they can solve my problem

**Comment by erichserrano** [comment]
Try emailing mycelium support directly. It me a week or so to hear back
from them, but they were able to help me.
